core/HomeButton.java

```diff
@@ -0,0 +1,46 @@
+package edu.asu.c3simulator.widgets;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+import edu.asu.c3simulator.screens.MainHub;
+//import edu.asu.c3simulator.screens.MainHub;
+/**
+ * This class creates a Home button actor, places it in the bottom left of the screen. Changes the screen to MainHub when clicked.
```

Comment:
Get rid of "This class"

Also, the class does not "create a Home button actor." It IS a HomeButton. The class javadoc should tell me what a HomeButton is.